### PR TITLE
Temp fix

### PR DESCRIPTION
**Pull request by hodge-jai**
### What this PR does (required):
- Fixes issue with protected routes where the components weren't rendering/loading properly on page refresh and manual url entry

### CODE REVIEW

File: client/src/components/EditProfile/EditPhoto/EditPhoto.tsx [hunk 1]
```diff
@@ -49,16 +49,24 @@ export default function EditPhoto(): JSX.Element {
...
               <Typography className={classes.reminder} component="h4" variant="h6">
                 Be sure to use a photo that clearly shows your face
               </Typography>
             </Box>
-            <Box display="flex" alignItems="center" justifyContent="center">
-              <Button className={classes.uploadButton} variant="outlined" color="primary" onClick={importData}>
+            <Box className={classes.centerRow}>
+              <Button component="label" className={classes.uploadButton} variant="outlined" color="primary">
                 Upload a file from your device
+                <input
```
Comment (sundayezeilo) [inline client/src/components/EditProfile/EditPhoto/EditPhoto.tsx:62]: Use MUI `TextField` component for this `<input>`

**Review by sundayezeilo — CHANGES_REQUESTED** [review]
Looks good! Please resolve the merge conflicts in this branch.

File: client/src/components/ProtectedRoute/ProtectedRoute.tsx [hunk 2]
```diff
@@ -1,11 +1,21 @@
+import { useState, useEffect } from 'react';
 import { Redirect, Route, RouteProps } from 'react-router';
 import { useAuth } from '../../context/useAuthContext';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { useHistory } from 'react-router-dom';
 
 export default function ProtectedRoute(routeProps: RouteProps) {
   const { loggedInUser } = useAuth();
-  if (loggedInUser) {
+  const [authenticated, setAuth] = useState<boolean>(false);
```
Comment (rajivtitus) [inline client/src/components/ProtectedRoute/ProtectedRoute.tsx:9]:
```suggestion
  const [authenticated, setAuthenticated] = useState<boolean>(false);
```
What is the reason for having a state here? The previous implementation of the protected route using the logged in user object (from AuthContext) accomplished the same in a simpler way :)

**Comment by rajivtitus** [comment]
Closing this PR as the protected route issue was fixed in another PR

**Pull request closed without merging**